integration-tests/docker/test_env/geth.go

```diff
@@ -29,13 +28,10 @@ const (
 
 type Geth struct {
 	EnvComponent
-	ExternalHttpUrl  string
-	InternalHttpUrl  string
-	ExternalWsUrl    string
-	InternalWsUrl    string
-	EthClient        *blockchain.EthereumClient
```

Comment:
@iljapavlovs why did you remove EthClient, ContractDeployer etc from this struct? Those are initialized in the methods of `Geth` so why put them outside?

Reply:
cc @skudasov